cpu/stm32/kconfigs/Kconfig.clk

```diff
@@ -85,15 +85,15 @@ config CLOCK_PLL_N
     default 90 if CPU_FAM_F4 && CLOCK_MAX_180MHZ
     default 216 if CPU_FAM_F7 && BOARD_HAS_HSE
     default 108 if CPU_FAM_F7
-    default 16 if CPU_FAM_WB
+    default 16 if CPU_FAM_WB || CPU_FAM_WL
```

Comment:
According to 

https://github.com/RIOT-OS/RIOT/blob/728aefa5418068368e821b0987030cb7cdbaa821/cpu/stm32/include/clk/l4l5wx/cfg_clock_default.h#L78-L82

it should be 12 in the case of `CPU_FAM_WL`

Reply:
Nice catch, fixup applied thanks.